### ISSUE TITLE
Can I set a border for the square area?

### ISSUE THREAD

**Issue by xuanxuandog** [ravi8x/Barcode-Reader#24]

**Comment by eternalBlast** [comment]
agree with this. Maybe can add the 4 corners for the border. Hi @ravi8x please consider this issue. Thanks.

**Comment by sivasankar0457** [comment]
Yes you can set the border to the camera in BarcodeReader class XML which is located in library file.
